internal/util/errors_test.go

```diff
@@ -64,8 +64,7 @@ func TestJoinErrors(t *testing.T) {
 	assertErrorIs(x, errFoo, false)
 	assertErrorIs(x, errBar, true)
 	s1 := "w{w{w{x}}: w{w{foo: bar}}}"
-	s2 := w1w2Xw2FooBar.Error()
-	if s1 != s2 {
+	if s2 := w1w2Xw2FooBar.Error(); s1 != s2 {
```

Comment:
Since two separate variables are used and the declaration of S2 is lengthy by itself, using one-liner here doesn't look so good IMO.

Reply:
It was done to satisfy `ifshort` linter and I think its fine the way it is.

Reply:
I agree with that too (maybe not her, but other places). But if that is the recommended coding style, I'll accept it. These changes don't make the lines extra long, and I'm sure I'll get used to reading the code this way.